guides/source/active_record_callbacks.md

```diff
@@ -837,58 +837,59 @@ lead to invalid data.
 [`upsert_all`]:
     https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert_all
 
-Suppressing Callbacks
----------------------
+Suppressing Saving
+------------------
 
-In certain scenarios, you may need to temporarily prevent certain callbacks from
-being executed within your Rails application. This can be useful when you want
-to skip specific actions during certain operations without permanently disabling
-the callbacks.
+In certain scenarios, you may need to temporarily prevent records from being saved within your Rails application.
+This can be useful to skip the callbacks that would run if the record is saved successfully.
 
-Rails provides a mechanism for suppressing callbacks using the
-[`ActiveRecord::Suppressor`
-module](https://api.rubyonrails.org/classes/ActiveRecord/Suppressor.html). By
-using this module, you can wrap a block of code where you want to suppress
-callbacks, ensuring that they are not executed during that specific operation.
+Rails provides a mechanism to prevent saving records using the [`ActiveRecord::Suppressor` module]
+(https://api.rubyonrails.org/classes/ActiveRecord/Suppressor.html).
+By using this module, you can wrap a block of code where you want to avoid saving records and running their save callbacks.
```

Comment:
Not running the callbacks of the suppressed class is just a side effect of not saving the records.
So I don't think we need to emphasis that:
```suggestion
By using this module, you can wrap a block of code where you want to avoid saving records of a specific type
that otherwise would be saved by the code block.
```